test/unittests/libdevcore/CommonJS.cpp

```diff
@@ -1,136 +1,128 @@
 /*
-	This file is part of cpp-ethereum.
+    This file is part of cpp-ethereum.
 
-	cpp-ethereum is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 3 of the License, or
-	(at your option) any later version.
+    cpp-ethereum is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
 
-	cpp-ethereum is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
+    cpp-ethereum is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
 
-	You should have received a copy of the GNU General Public License
-	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+    You should have received a copy of the GNU General Public License
+    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 */
-/** @file CommonJS.cpp
- * @author Lefteris Karapetsas <lefteris@refu.co>
- * @date 2015
- * Tests for functions in CommonJS.h
- */
 
-#include <boost/test/unit_test.hpp>
 #include <libdevcore/CommonJS.h>
-#include <test/tools/libtesteth/TestOutputHelper.h>
+
+#include <gtest/gtest.h>
 
 using namespace dev;
 using namespace std;
-using namespace test;
-
-BOOST_FIXTURE_TEST_SUITE(CommonJSTests, TestOutputHelperFixture)
 
-BOOST_AUTO_TEST_CASE(test_toJS)
+TEST(common_js, toJS)
```

Comment:
Maybe let's try to use camelCase more consistently for both test case names and test names, so this is perhaps `CommonJS`